### PR TITLE
feat(presentational): add Subtitle component

### PR DESCRIPTION
**Pull request by agory**

### CODE REVIEW

File: src/web/components/Subtitle/index.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export default function Subtitle({ children, level }) {
+  const classes = `subtitle${` is-${level + 1}`}`;
```
Comment (mfrachet) [inline src/web/components/Subtitle/index.js]: Why not to simply 

```javascript
`subtitle is-${level + 1}`
```

File: src/web/components/Subtitle/index.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  const Heading = `h${level + 1}`;
```
Comment (mfrachet) [inline src/web/components/Subtitle/index.js]:
```javascript
level + 1
```

is made two times inside this component. Maybe we could make it only once ?

Comment (mfrachet) [reply src/web/components/Subtitle/index.js]: Is this working ?

**Comment by mfrachet** [comment]
Now we've included storybook, you can directly add a storybook while creating a PR for a component, will be easier to review

File: src/web/components/Subtitle/index.js (hunk 2 above)
Comment (agory) [reply src/web/components/Subtitle/index.js]: Yes, it works :smile: .
Initially I used a switch case for the tests, but I couldn't get 100% coverage without doing a test per case. So I did a little research on Stackoverflow and I found this solution.

I'm gonna refactor level.

File: src/web/components/Subtitle/index.js [hunk 3]
```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export default function Subtitle({ children, level }) {
+  const raisedLevel = level + 1;
+  const classes = `subtitle is-${raisedLevel}`;
+  const Heading = `h${raisedLevel}`;
+  return <Heading className={classes}>{children}</Heading>;
```
Comment (mfrachet) [inline src/web/components/Subtitle/index.js]: No error on eslint at that line ?

Comment (agory) [reply src/web/components/Subtitle/index.js]: Nop for me.

Comment (agory) [reply src/web/components/Subtitle/index.js]: Yes, it works smile .
Initially I used a switch case for the tests, but I couldn't get 100% coverage without doing a test per case. So I did a little research on Stackoverflow and I found this solution.

I'm gonna refactor level.

**Pull request merged**